From fc79d0beffc72950292e5adbe5e3481664e689d8 Mon Sep 17 00:00:00 2001
From: Allen Martin <armartin@gmail.com>
Date: Sun, 5 Feb 2023 20:18:50 -0800
Subject: [PATCH 25/31] riscv: dts: bflb: ox64: Fix address/size-cells

The number of address cells needed here (one) does not match the
implicitly-defined default number of cells.

Signed-off-by: Allen Martin <armartin@gmail.com>
---
 arch/riscv/boot/dts/bouffalolab/bl808-pine64-ox64.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/riscv/boot/dts/bouffalolab/bl808-pine64-ox64.dts b/arch/riscv/boot/dts/bouffalolab/bl808-pine64-ox64.dts
index f9ed636291f6..f1db126d88e7 100644
--- a/arch/riscv/boot/dts/bouffalolab/bl808-pine64-ox64.dts
+++ b/arch/riscv/boot/dts/bouffalolab/bl808-pine64-ox64.dts
@@ -34,6 +34,8 @@ xip_flash@58500000 {
 		linux,mtd-name = "xip-flash.0";
 		erase-size = <0x10000>;
 		bank-width = <4>;
+		#address-cells = <1>;
+		#size-cells = <1>;
 	};
 };
 
-- 
2.25.1

